src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -179,5 +179,9 @@ internal class CommonLocalizableStrings
         public const string SpecifiedNameExists = "Specified name {0} already exists. Please specify a different name.";
         public const string SpecifiedAliasExists = "Specified alias {0} already exists. Please specify a different alias.";
         public const string MandatoryParameterMissing = "Mandatory parameter {0} missing for template {1}. ";
+
+        public const string ProjectNotCompatibleWithFramework = "Project `{0}` is not compatible with `{1}`.";
```

Comment:
I thought I had some comments on this string before. Don't know where it went, but I believe this needs to be improved. Just the message that A is not compatible with B is very un-actionable. There is not hint as to what I need to do to fix that or in what sense they are incompatible.